src/main/resources/logback.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why you removed this file?

Reply:
We don't need to set logback as the root project level.
Because every scenario is not a submodule. It is it's own Gradle project.